datasource/src/test/scala/astraea/spark/rasterframes/datasource/rastersource/RasterSourceDataSourceSpec.scala

```diff
@@ -108,6 +113,16 @@ class RasterSourceDataSourceSpec extends TestEnvironment with TestData {
       df.schema.size should be(2)
     }
 
+    it("should read a multiple files from a table") {
+      Seq(cogPath, l8samplePath, nonCogPath).toDF("path").createOrReplaceTempView("pathsTable")
+      val df = spark.read.rastersource
+        .fromTable("pathsTable", "path")
+        .withTileDimensions(128, 128)
```

Comment:
So is this coercing a 128 x 128 tile size on the data that is read from each path?

Reply:
Yes, discretizing all scenes to this common size.

Reply:
The RasterFrames default is 256x256.